scripts/minikube.sh

```diff
@@ -99,6 +98,19 @@ function minikube_losetup() {
     ${minikube} ssh 'sudo sh -c "rm -f /sbin/losetup && cp ~docker/losetup /sbin"'
 }
 
+function minikube_supports_psp() {
```

Comment:
please update the commit message why the reordering is required

Reply:
Yep. Until then:

Generally, we define all the functions at the top and start the runnable commands (entry points to script start).
Only this function is odd in the script unlike the rest of them taking away the readability.